native.js

```diff
@@ -405,7 +405,7 @@ Native["java/lang/Class.newInstance1.(Ljava/lang/Object;)V"] = function(addr, oA
   if (!methodInfo) {
     throw $.newInstantiationException("Can't instantiate classes without a nullary constructor");
   }
-  J2ME.getLinkedMethod(methodInfo).call(o);
+  J2ME.getLinkedMethod(methodInfo)(oAddr);
```

Comment:
We should file a follow to clean up the natives that use getHandle only to get the klass property of the object (and use klassIdMap instead).